essentials-module-dynmap/src/main/kotlin/io/github/nyliummc/essentials/entities/FabricDynmapServer.kt

```diff
@@ -0,0 +1,156 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) NyliumMC
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package io.github.nyliummc.essentials.entities
+
+import net.minecraft.block.SignBlock
+import net.minecraft.server.MinecraftServer
+import net.minecraft.text.LiteralText
+import net.minecraft.util.Identifier
+import net.minecraft.util.math.BlockPos
+import net.minecraft.util.registry.Registry
+import net.minecraft.world.dimension.DimensionType
+import org.dynmap.DynmapChunk
+import org.dynmap.DynmapWorld
+import org.dynmap.common.DynmapListenerManager
+import org.dynmap.common.DynmapPlayer
+import org.dynmap.common.DynmapServerInterface
+import org.dynmap.utils.MapChunkCache
+import java.util.HashSet
+import java.util.concurrent.Callable
+import java.util.concurrent.Future
+
+
+class FabricDynmapServer(private val minecraftServer: MinecraftServer) : DynmapServerInterface() {
+    override fun scheduleServerTask(run: Runnable?, delay: Long) {
+        System.out.printf("Task requested with delay %d%n", delay)
+    }
+
+    override fun <T> callSyncMethod(task: Callable<T>?): Future<T>? {
+        println("Sync method requested")
+        return null
+    }
+
+    override fun getOnlinePlayers(): Array<DynmapPlayer?>? {
+        return minecraftServer.playerManager.playerList.map(::FabricDynmapOnlinePlayer).toTypedArray()
+    }
+
+    override fun reload() {
+        TODO("Not yet implemented")
```

Comment:
What does this correspond on on other implementions?

Reply:
A full plugin reload on most, unsure about the details.

Reply:
If there is a sponge impl of dynmap, take a look at what they do there